server/controller/rateLimit.js

```diff
@@ -0,0 +1,16 @@
+const redisClient = require("../config/redis");
+
+module.exports.rateLimit = async (req, res, next) => {
```

Comment:
Don't directly export create a variable and then export using module.exports as you done in redis.js config file